openvino_xai/explainer/explanation.py

```diff
@@ -149,6 +150,72 @@ def save(self, dir_path: Path | str, name: str | None = None) -> None:
             image_name = f"{save_name}_target_{target_name}.jpg" if save_name else f"target_{target_name}.jpg"
             cv2.imwrite(os.path.join(dir_path, image_name), img=map_to_save)
 
+    def plot(self, targets: np.ndarray | List[int | str] | None = None, backend="matplotlib") -> None:
+        """
+        Plots saliency maps using the specified backend.
+
+        This function plots available saliency maps using the specified backend. Additionally, target classes
+        can be specified by passing a list or array of target class indices or names. If a provided class is
+        not available among the saliency maps, it is omitted.
+
+        Args:
+            targets (np.ndarray | List[int | str] | None): A list or array of target class indices or names to plot.
+                By default, it's None, and all available saliency maps are plotted.
+            backend (str): The plotting backend to use. Can be either 'matplotlib' (recommended for Jupyter)
+                or 'cv' (recommended for Python scripts). Default is 'matplotlib'.
+        """
+
+        if targets is None or explains_all(targets):
+            class_indices = self.targets
+        else:
+            target_indices = get_target_indices(targets, self.label_names)
+            class_indices = [cls_idx for cls_idx in target_indices if cls_idx in self.targets]
```

Comment:
What about warning in the case when specified target not in self.targets?

Reply:
`cls_idx` -> `target_index`, for better generalization, especially considering upcoming detection support of per-box